app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -24,8 +24,8 @@ function initializeBaseTracking() {
       }
     }, 25);
     eventListening();
-    trackCustomImpressions();
   }
+  trackCustomImpressions();
```

Comment:
Is there a reason why you moved [this](https://github.com/forem/forem/blob/main/app/assets/javascripts/initializers/initializeBaseTracking.js.erb#L27) down a line?

Reply:
Yes.  I moved the function call outside of an `if` conditional.  That conditional would only fire if the forem instance had Google Analytics configured.